R-package/src/graph_spme.cpp

```diff
@@ -0,0 +1,42 @@
+// Main functions for GraphSPME
+// License: GPL-3
+
+#include <Rcpp.h>
+#include "graph_spme.hpp"
+
+// Enable C++11 via this plugin (Rcpp 0.10.3 or later)
+// [[Rcpp::plugins("cpp11")]]
+
+/*
+ * Covariance shrinkage estimate as specified in Touloumis (2015)
+ */
+// [[Rcpp::export(.cov_shrink_spd)]]
+Eigen::Matrix<double,Eigen::Dynamic,Eigen::Dynamic> _cov_shrink_spd(
+        Eigen::Matrix<double,Eigen::Dynamic,Eigen::Dynamic>& x
+){
+    return cov_shrink_spd(x);
+}
```

Comment:
This is not super elegant, but `Rcpp` requires one graph_spme.cpp file under /src to properly build the R-package, including among other things `RcppExports.cpp`. `Rcpp` will look for the `Rcpp` attributes of type `// [[Rcpp::export]]` in graph_spme.cpp

Reply:
Had a feeling that it would be required yes! Wondering why it built fine on my machine without it though :thinking:

Reply:
With me `R CMD build R-package` worked, but not `R CMD install` on the resulting tar package.
Also, building and installing using RStudio deleted the `RcppExports.cpp` because there were no .cpp file.